static/solo-cta/js/mobile/modal/index.js

```diff
@@ -0,0 +1,296 @@
+(function() {
+
+    var body = document.body;
+    var id = body.getAttribute("data-id") || "";
+    var config = window.parent.__TVPage__.config[id];
+    var eventPrefix = config.events.prefix;
+    var mainEl = Utils.getById(id);
```

Comment:
At this moment, the global `Utils` is not yet ready loaded, please do either:

`var mainEl = document.getElementById(id)`

or

`var mainEl;`

and the you update the value in the `initialize` function